api/sixtyfps-rs/migration.md

```diff
@@ -56,6 +56,22 @@ fn model_tracker(&self) -> &dyn ModelTracker {
 }
 ```
 
+#### `Option<SharedModel>` replaces `ModelHandle`
```

Comment:
This doesn't seem quite right, because in < 0.2.0 there's no `Option<SharedModel>`. I think perhaps you meant "ModelHandle was renamed to SharedModel"?

In that case it's a simple search and replace, if we don't change the API in a way that breaks existing code, just add new convenience (like the `From` from the model without Rc).

Reply:
> if we don't change the API in a way that breaks existing code

We will if we remove the Deref to the Option<Rc> and the pub access to .0, which we should.